### DISCUSSION TITLE
GraphAPI Get Message Alias

### DISCUSSION THREAD

**Discussion by swordfish291** [microsoftgraph/msgraph-sdk-dotnet#2581] [Q&A]
Hello Team,

I am implementing a solution using GraphAPI to extract and categorize attachments sent to a mailbox that has many aliases configured. I want to categorize the messages based on the alias these are sent to but I am not able to find the alias property in the return message. I am using a get request:

GET https://graph.microsoft.com/beta/me/messages/{message-id}

How can I retrieve the alias information? Is there any other way to retrieve this if its not possible using GraphAPI?

Thank you

Regards
Abdul

**Comment by andrueastman** [comment]
Thanks for raising this @swordfish291 

Any chance you can ask the question at the link below to get feedback from the API owners? We may not be best placed to answer this as this repo mainly deals with SDK related issues.

https://aka.ms/askGraph
